### PR TITLE
Adding CNAME to examples/web folder to make this config persistent

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Adding the CNAME file to the examples web folder, so we don't need to recreate it every time that the examples is deployed.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
- [ ] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
- [ ] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
- [x] I read and followed the [Flame Style Guide].
- [ ] I have added a description of the change under `[next]` in `CHANGELOG.md`.
- [x] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Review by spydon — APPROVED** [review]

**Review by luanpotter — APPROVED** [review]
LGTM. is this a github bug where it creates the file on the wrong place via the UI?

File: examples/web/CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+examples.flame-engine.org
```
Comment (luanpotter) [inline examples/web/CNAME:1]: does it need https here?

Comment (erickzanardo) [reply examples/web/CNAME:1]: nope, this is just the  domain

**Comment by erickzanardo** [comment]
> LGTM. is this a github bug where it creates the file on the wrong place via the UI?

Nope, it create in the correct page, but since the site is deployed automatically, and we replace the old site with the new one, we need that file our source folder, otherwise every new deploy will happen without the site.

**Pull request merged**